Advanced-Recruit-Training/README.adoc

```diff
@@ -1,27 +1,36 @@
-# U of C Solar Car Advanced Recruit Training
+= U of C Solar Car Advanced Recruit Training
 Intro project to learn c++ and QT
 
+== Running the project:
+
+=== Via Command Line
+
 To build this project on linux navigate to the source directory and run the following commands:
 
 `qmake`
 
 `make`
 
-To run the project:
-`./batteryLife` - Refer to help for further instructions
+`./batteryLife` - Refer to help for further instructions on how to add arguments to the program.
 
+=== Via QT Creator
 In QT Creator, the above `qmake` and `make` are located in `Build`, and the project can be run through the play button. You'll have to set the appropriate command line arguments in `Projects > Build & Run > Run`. 
 
+== Objective
+
 The goal of this project is to give new recuits a good starting point to learn the required skills to work on the other projects. 
-If you ever need any help with how to use Qt's functions, a good start would be to look at the [Qt's reference manual](http://doc.qt.io/qt-5.6/reference-overview.html)
+If you ever need any help with how to use Qt's functions, a good start would be to look at http://doc.qt.io/qt-5.6/reference-overview.html[Qt's reference manual]
```

Comment:
I don't think we're on version 5.6, should it be referencing 5.5?

Reply:
There's only documentation for Qt 5.9, 5.6, and 4.8 :/
http://doc.qt.io/

Reply:
Sad times